flake.nix

```diff
@@ -27,8 +27,84 @@
                         base
                         wai
                         text
+
+                        # Uncomment on local development for testing
+                        # hspec
                     ];
                 };
+
+                # Custom configuration that will start with `devenv up`
+                devenv.shells.default = {
+                    # Start Mailhog on local development to catch outgoing emails
+                    services.mailhog.enable = true;
+
+                    # Custom processes that don't appear in https://devenv.sh/reference/options/
+                    processes = {
+                        # Uncomment if you use tailwindcss.
+                        # tailwind.exec = "tailwindcss -c tailwind/tailwind.config.js -i ./tailwind/app.css -o static/app.css --watch=always";
+                    };
+                };
+            };
+
+            # Adding the new NixOS configuration for "qa"
```

Comment:
Can we add a link to https://ihp.digitallyinduced.com/Guide/deployment.html#deploying-with-deploytonixos here? So people have access to more context

Reply:
```suggestion
            # Adding the new NixOS configuration for "qa"
            # See https://ihp.digitallyinduced.com/Guide/deployment.html#deploying-with-deploytonixos for more info
```